tests/UI/campaigns/functional/BO/03_catalog/04_attributesAndFeatures/01_attributes/02_changePosition.ts

```diff
@@ -0,0 +1,178 @@
+// Import utils
+import basicHelper from '@utils/basicHelper';
+import helper from '@utils/helpers';
+import testContext from '@utils/testContext';
+
+// Import commonTests
+import loginCommon from '@commonTests/BO/loginBO';
+
+// Import pages
+import dashboardPage from '@pages/BO/dashboard';
+import attributesPage from '@pages/BO/catalog/attributes';
+import viewAttributePage from '@pages/BO/catalog/attributes/view';
+
+// Import data
+import Attributes from '@data/demo/attributes';
+
+import {expect} from 'chai';
+import type {BrowserContext, Page} from 'playwright';
+
+const baseContext: string = 'functional_BO_catalog_attributesAndFeatures_attributes_changePosition';
+
+/*
+Scenario:
+- Go to attributes page
+- Sort by position ASC
+- Change first attribute position to 3
+- Reset attribute position
+- Filter by attribute Color
+- View attribute
+- Change first value position to 3
+- Reset value position
+ */
+describe('BO - Catalog - Attributes & Features : Change attributes & values position', async () => {
+  let browserContext: BrowserContext;
+  let page: Page;
+
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to \'Catalog > Attributes & Features\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToAttributesPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.catalogParentLink,
+      dashboardPage.attributesAndFeaturesLink,
+    );
+
+    const pageTitle: string = await attributesPage.getPageTitle(page);
```

Comment:
```suggestion
    const pageTitle = await attributesPage.getPageTitle(page);
```

You don't need to type our return values from our methods.

@NeOMakinG called that the `type inference`.

https://www.typescriptlang.org/docs/handbook/type-inference.html